### PR TITLE
Adding discard cards

### PR DESCRIPTION
**Pull request by gpfaia**

### CODE REVIEW

File: backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_corner_store.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from backend.src.game.newcards.discard_cards.victory_point_manipulation_cards.corner_store import CornerStore
+
+
+def test_commuter_train_adds_1_victory_points(player):
```
Comment (borisv13) [inline backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_corner_store.py]: The test names say commuter_train lol

File: backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_evacuation_orders.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+def test_evacuation_orders_subtracts_5_victory_points(five_players):
+    for player in five_players:
+        player.update_victory_points_by(10)
+    EvacuationOrders().immediate_effect(None, five_players)
+    assert all(other_players.victory_points ==
+               5 for other_players in five_players)
+
+
+def test_apartment_building_costs_7_energy():
```
Comment (borisv13) [inline backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_evacuation_orders.py]: test name again

File: backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_evacuation_orders.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from backend.src.game.newcards.discard_cards.victory_point_manipulation_cards.evacuation_orders import \
+    EvacuationOrders
+
+
+def test_evacuation_orders_subtracts_5_victory_points(five_players):
+    for player in five_players:
+        player.update_victory_points_by(10)
+    EvacuationOrders().immediate_effect(None, five_players)
+    assert all(other_players.victory_points ==
+               5 for other_players in five_players)
```
Comment (borisv13) [inline backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_evacuation_orders.py]: Would it be good to have a test case that asserts that VP doesn't go below 0? Might not be this classes' responsibility, but that test needs to exist somewhere.

**Comment by gpfaia** [comment]
This was before the move, will resubmit on the new PR

**Pull request closed without merging**